metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorConfig.java

```diff
@@ -0,0 +1,180 @@
+/*
+ *       Copyright 2017 Netflix, Inc.
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ */
+
+package com.netflix.metacat.connector.hive;
+
+import com.netflix.metacat.common.server.util.DataSourceManager;
+import com.netflix.metacat.common.server.util.MetacatConnectorProperties;
+import com.netflix.metacat.connector.hive.client.DefaultNoOpHiveClient;
+import com.netflix.metacat.connector.hive.converters.HiveConnectorInfoConverter;
+import com.netflix.metacat.connector.hive.converters.HiveTypeConverter;
+import com.netflix.metacat.common.server.util.JdbcUtil;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+/**
+ * Hive configs.
+ *
+ * @author zhenl
+ * @since 1.1.0
+ */
+@Configuration
+public class HiveConnectorConfig {
+
+    /**
+     * create hive connector partition service.
+     *
+     * @param catalogName          catelog name
+     * @param metacatHiveClient    hive client
+     * @param hiveMetacatConverter metacat converter
+     * @return HiveConnectorPartitionService
+     */
+    @Bean(name = "hivePartitionService")
```

Comment:
Is there a reason you're explicitly naming the beans as opposed to just relying on the method names? There should only be one partition service right?